src/extension/extension.ts

```diff
@@ -512,16 +502,16 @@ export async function activate(ctx: sourcegraph.ExtensionContext): Promise<void>
                     logger.log('Hover trace', span.generateTraceURL())
                 }
 
-                const lsifResult = await lsif.hover(textDocument, position)
-                if (lsifResult) {
-                    yield lsifResult.value
-                } else if (!config.value['typescript.serverUrl']) {
-                    const result = await basicCodeIntel.hover(textDocument, position)
-                    if (result) {
-                        yield { ...result, badge: impreciseBadge }
-                    } else {
-                        yield undefined
-                    }
+                // const lsifResult = await lsif.hover(textDocument, position)
```

Comment:
Will clean up this and additional dead code in a cleanup PR.

Reply:
Why should that not be done in this PR?

Reply:
I can but after the initial review. Removing this completely made a horrendous diff.

Reply:
I must admit I have some trouble understanding how this all fits together, and having the usage just be a diff that comments out code doesn't help unfortunately. I think it may actually be better to remove for review purposes, but it's hard for me to say without fully understanding everything yet

Reply:
Pushed!